src/assets/top.css

```diff
@@ -37,15 +37,14 @@ footer, header, hgroup, main, menu, nav, section {
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     text-align: center;
-    color: #50312c;
+    color: #f2e6e5;
```

Comment:
I was imagining you'd create a different css file for the book, and leave top for my debugging test apps.

can you move this file to something like "room.css" and revert top.css to what it was?

Reply:
I did this in my branch. Should I submit a new pull request? See my questions in Teams.